dev-libs/bitshuffle/bitshuffle-0.3.5-r1.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021-2022 Gentoo Authors
+# Copyright 2021-2024 Gentoo Authors
```

Comment:
Please include [kiyo-masui/bitshuffle@`d29228f` (#140)](https://github.com/kiyo-masui/bitshuffle/pull/140/commits/d29228fae82b51dec5bddc92c9b2c8ac1a0e402d) as a patch to fix https://bugs.gentoo.org/904356.

Reply:
> Please include [kiyo-masui/bitshuffle@`d29228f` (#140)](https://github.com/kiyo-masui/bitshuffle/pull/140/commits/d29228fae82b51dec5bddc92c9b2c8ac1a0e402d) as a patch to fix https://bugs.gentoo.org/904356.

```
patching file src/bitshuffle_core.c
Hunk #1 succeeded at 43 (offset -6 lines).
Hunk #2 FAILED at 658.
Hunk #3 succeeded at 768 (offset -29 lines).
Hunk #4 FAILED at 832.
Hunk #5 succeeded at 1102 (offset -45 lines).
Hunk #6 succeeded at 1118 (offset -45 lines).
2 out of 6 hunks FAILED -- saving rejects to file src/bitshuffle_core.c.rej
```

we'd better try that separately

Reply:
OK, I'll handle that once this is in.

Reply:
> OK, I'll handle that once this is in.

renaming `snappy-java` can be done later together with vbump